packages/ui/src/views/project/project.types.ts

```diff
@@ -0,0 +1,19 @@
+export interface MemberData {
+  display_name: string
+  role: string
+  email: string
+  avatarUrl: string
+  timestamp: string
+  uid: string
+}
+
+export interface IMemberListStore {
+  // state
```

Comment:
nit: we don't need state/actions comments

Reply:
Will cleanup

Reply:
Done in https://github.com/harness/canary/pull/613.